### PR TITLE
Update nosql_find_injection to support ES6 syntax

### PR DESCRIPTION
**Pull request by CharlyJazz**
There is a lot of backend software running in ES6 syntax. And this syntax is "import X from "Y" replacing the old const X = require("Y")

### CODE REVIEW

File: njsscan/rules/semantic_grep/database/nosql_find_injection.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@ rules:
 - id: node_nosqli_injection
   patterns:
   - pattern-not-inside: |
+      import $SANITIZE from 'mongo-sanitize'
```
Comment (ajinabraham) [inline njsscan/rules/semantic_grep/database/nosql_find_injection.yaml:5]: You need to move this to a separate `pattern-not-inside:`

```
  - pattern-not-inside: |
        import $SANITIZE from 'mongo-sanitize'
        ... 
        $SANITIZE(...)
         ...
  ```

**Review by ajinabraham — CHANGES_REQUESTED** [review]
Thanks for the PR. ❤️ 
Please see review comments.

File: njsscan/rules/semantic_grep/database/nosql_find_injection.yaml (hunk 1 above)
Comment (CharlyJazz) [reply njsscan/rules/semantic_grep/database/nosql_find_injection.yaml:5]: in the same file?

Comment (ajinabraham) [reply njsscan/rules/semantic_grep/database/nosql_find_injection.yaml:5]: Yes under `patterns`

Comment (CharlyJazz) [reply njsscan/rules/semantic_grep/database/nosql_find_injection.yaml]: Can you check again? Greetings

**Review by ajinabraham — APPROVED** [review]

**Pull request merged**